app/addons/documents/tests/changes.componentsSpec.react.jsx

```diff
@@ -179,7 +188,185 @@ define([
+      { id: 'doc_1', seq: 4, deleted: false, changes: { code: 'here' } },
+      { id: 'doc_2', seq: 1, deleted: false, changes: { code: 'here' } },
+      { id: 'doc_3', seq: 6, deleted: true, changes: { code: 'here' } },
+      { id: 'doc_4', seq: 7, deleted: false, changes: { code: 'here' } },
+      { id: 'doc_5', seq: 1, deleted: true, changes: { code: 'here' } }
+    ]);
+
+    beforeEach(function () {
+      Actions.setChanges({
+        changes: changeList.models,
+        filters: [],
+        databaseName: 'testDatabase'
+      });
+      headerEl  = TestUtils.renderIntoDocument(<Changes.ChangesHeaderController />, containerEl);
+      changesEl = TestUtils.renderIntoDocument(<Changes.ChangesController />, containerEl);
+    });
+
+    afterEach(function () {
+      Stores.changesStore.reset();
+      React.unmountComponentAtNode(containerEl);
+    });
+
+
+    it('should list the right number of changes', function () {
+      assert.equal(changeList.length, $(changesEl.getDOMNode()).find('.change-box').length);
+    });
+
 
+    it('"false"/"true" filter strings should apply to change deleted status', function () {
+      // expand the header
+      var $headerEl = $(headerEl.getDOMNode());
+      TestUtils.Simulate.click($(headerEl.getDOMNode()).find('a')[0]);
+
+      // add a filter
+      var addItemField = $headerEl.find('.js-changes-filter-field')[0];
+      var submitBtn = $headerEl.find('[type="submit"]')[0];
+      addItemField.value = 'true';
+      TestUtils.Simulate.change(addItemField);
+      TestUtils.Simulate.submit(submitBtn);
+
+      // confirm only the two deleted items shows up and the IDs maps to the deleted rows
+      assert.equal(2, $(changesEl.getDOMNode()).find('.change-box').length);
+      assert.equal('doc_3', $(changesEl.getDOMNode()).find('.js-doc-id')[0].innerHTML);
+      assert.equal('doc_5', $(changesEl.getDOMNode()).find('.js-doc-id')[1].innerHTML);
+    });
+
+
+    it('confirms that a filter affects the actual search results', function () {
+      // expand the header
+      var $headerEl = $(headerEl.getDOMNode());
+      TestUtils.Simulate.click($(headerEl.getDOMNode()).find('a')[0]);
+
+      // add a filter
+      var addItemField = $headerEl.find('.js-changes-filter-field')[0];
+      var submitBtn = $headerEl.find('[type="submit"]')[0];
+      addItemField.value = '6'; // should match doc_3's sequence ID
+      TestUtils.Simulate.change(addItemField);
+      TestUtils.Simulate.submit(submitBtn);
+
+      // confirm only one item shows up and the ID maps to what we'd expect
+      assert.equal(1, $(changesEl.getDOMNode()).find('.change-box').length);
+      assert.equal('doc_3', $(changesEl.getDOMNode()).find('.js-doc-id')[0].innerHTML);
+    });
+
+
+    // confirms that if there are multiple filters, ALL are applied to return the subset of results that match
+    // all filters
+    it('multiple filters should all be applied to results', function () {
```

Comment:
For future reference this test and the test before it you could actually test at the store level since the views just render what ever changes they are given. The way you have done it here is more like an integration test than a unit test.